### ISSUE TITLE
Battered Junkbox and I assume a lot of lockboxes can have nothing inside it

### ISSUE THREAD

**Issue by EdyGlockenspiel** [gamefreedomgit/Maelstrom#2839]
**Description:** This problem seems to be common on private servers, openable items and even some times chests in world can have a chance to obtain nothing. This is obviously not intended.

**How to reproduce:** Unlock Battered Junkbox and open them. Eventually one will have nothing in it after just a few.

**How it should work:** When you open any container type item it should pick from a pool of items to give you, so that you get *something* 100% of the time.

**Database links:**
https://cata-twinhead.twinstar.cz/?item=16882#contains as you can see it can drop quite a few things, and it might be using the literal  % drop on them instead of pick one with higher % items being picked that much more often ect.

**Comment by dantethehunter** [comment]
I agree and i can also say that crates and boxes from fishing wreckages the most of the times do not obtain items, or something like 5 copper grey items.

**Comment by EdyGlockenspiel** [comment]
I have never experienced this on retail servers yet almost every pserver I have played on his this same problem and I have no idea why, it is rather annoying.  Wish there was an easy fix all for all lootable items/chests like this. But I assume they would have to go and look at each and every single item/chest individually.

**Comment by EdyGlockenspiel** [comment]
https://cata-twinhead.twinstar.cz/?item=20768#contains Oozing Bag as well.

**Comment by EdyGlockenspiel** [comment]
https://www.wowhead.com/item=43575/reinforced-junkbox This too. It should have an average of about 3s 75c in 'raw gold' per box 100% of the time on top of it always containing items but does not.

**Comment by EdyGlockenspiel** [comment]
https://cata-twinhead.twinstar.cz/?item=27513#contains This too. It should have one of the select mats in side like a stack of leather or a stack of ore, and then on top of that a chance to contain the book as well.

**Comment by EdyGlockenspiel** [comment]
https://www.wowhead.com/item=16885/heavy-junkbox
https://cata-twinhead.twinstar.cz/?item=16885 Heavy Heavy Junkbox, too. It should have an average of about 3s 75~ copper inside it 100% of the time as well.

**Comment by EdyGlockenspiel** [comment]
https://www.wowhead.com/item=29569/strong-junkbox Strong Junkbox as well. It should have about 3s 75c in it on average every time, on top of it's items it drops.

**Comment by EdyGlockenspiel** [comment]
Khorium Lockbox as well. https://cata-twinhead.twinstar.cz/?item=31952#contains

**Comment by Afgann** [comment]
fixed
